packages/space-opera/src/test/animation_controls/animation_controls_test.ts

```diff
@@ -26,13 +27,13 @@ import {Dropdown} from '../../components/shared/dropdown/dropdown.js';
 import {dispatchReset} from '../../reducers.js';
 import {reduxStore} from '../../space_opera_base.js';
 
-const ANIMATED_GLB_PATH = '../base/shared-assets/models/RobotExpressive.glb';
+const ANIMATED_GLB_PATH = 'packages/shared-assets/models/RobotExpressive.glb';
 
-describe('animation controls test', () => {
+suite('animation controls test', () => {
   let preview: ModelViewerPreview;
   let animationControls: AnimationControls;
 
-  beforeEach(async () => {
+  setup(async () => {
```

Comment:
We inherited the space-opera code base from another team that used `jasmine` as their test framework instead of `mocha`. I did a find-replace and then some more re-jiggering to get it all updated to be consistent with our other packages.